docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -522,10 +524,10 @@ Deploys single instance Elastic Agent Deployment in Fleet mode with APM integrat
 [id="{p}-elastic-agent-fleet-known-limitations"]
 == Known limitations
 
-=== Running as root and within a single namespace
-Elastic Agent in Fleet mode has to run as root, and in the same namespace as the Elasticsearch cluster it connects to.
+=== Running as root and within a single namespace (ECK < 2.8.0 and Agent < 7.14.0)
+Until version 7.14.0 and ECK 2.8.0, Elastic Agent in Fleet mode has to run as root and in the same namespace as the Elasticsearch cluster it connects to. Since ECK 2.8.0 Agent in fleet mode is required to be greater than version 7.14.0. Both the Fleet Server and the Elastic Agent are not required to run as root but there are some known issues when <<{p}_storing_local_state_in_host_path_volume>> that must be noted.
```

Comment:
```suggestion
Until version 7.14.0 and ECK 2.8.0, Elastic Agent in Fleet mode has to run as root and in the same namespace as the Elasticsearch cluster it connects to. Since ECK 2.8.0, Agent in Fleet mode is required to be greater than version 7.14.0. Both the Fleet Server and the Elastic Agent are not required to run as root but there are some known issues when <<{p}_storing_local_state_in_host_path_volume>> that must be noted.
```